ipatests/test_integration/test_upgrade.py

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+"""
+Module provides tests to verify that the upgrade script works.
+"""
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_plugins.integration import tasks
+from ipaserver.install import sysupgrade
```

Comment:
Unused import.

Reply:
Sigh, it's what I get for rushing. Fixed.